spec/helpers/middleman/page_spec.rb

```diff
@@ -187,12 +187,12 @@
     end
 
     it "should respond with a localized 'landing_page' based page" do
-      expect(@app.locale_page(page: 'about-sharesight')[:page_title]).to eq('About Us | Sharesight')
-      expect(@app.locale_page(page: 'about-sharesight', locale_obj: @app.get_locale_obj('global'))[:page_title]).to eq('About Us | Sharesight')
-      expect(@app.locale_page(page: 'about-sharesight', locale_obj: @app.get_locale_obj('au'))[:page_title]).to eq('About Us | Sharesight Australia')
-      expect(@app.locale_page(page: 'about-sharesight', locale_obj: @app.get_locale_obj('ca'))[:page_title]).to eq('About Us | Sharesight Canada')
-      expect(@app.locale_page(page: 'about-sharesight', locale_obj: @app.get_locale_obj('nz'))[:page_title]).to eq('About Us | Sharesight New Zealand')
-      expect(@app.locale_page(page: 'about-sharesight', locale_obj: @app.get_locale_obj('uk'))[:page_title]).to eq('About Us | Sharesight UK')
+      expect(@app.locale_page(page: 'pricing')[:page_title]).to eq('About Us | Sharesight')
+      expect(@app.locale_page(page: 'pricing', locale_obj: @app.get_locale_obj('global'))[:page_title]).to eq('About Us | Sharesight')
+      expect(@app.locale_page(page: 'pricing', locale_obj: @app.get_locale_obj('au'))[:page_title]).to eq('About Us | Sharesight Australia')
+      expect(@app.locale_page(page: 'pricing', locale_obj: @app.get_locale_obj('ca'))[:page_title]).to eq('About Us | Sharesight Canada')
+      expect(@app.locale_page(page: 'pricing', locale_obj: @app.get_locale_obj('nz'))[:page_title]).to eq('About Us | Sharesight New Zealand')
+      expect(@app.locale_page(page: 'pricing', locale_obj: @app.get_locale_obj('uk'))[:page_title]).to eq('About Us | Sharesight UK')
```

Comment:
Oh ye this is where the test is failing - title mismatches. Might be fine to just delete these though as Kylor mentioned that the test coverage here isn't that important... up to you!

Reply:
awesome thanks! might as well remove them if they're not needed